plugins/notify.py

```diff
@@ -0,0 +1,11 @@
+from will.plugin import WillPlugin
+from will.decorators import respond_to, periodic, hear, randomly, route, rendered_template
+
+
+class NotifyPlugin(WillPlugin):
+
+    @route("/notify/<build_id>/<text>")
+    def send_notification(self, build_id, text):
+        notification_list = self.load("notify_{}".format(build_id), '')
```

Comment:
I figured it could be used more than once.  Setting a lifecycle rule on the redis keys would be a good idea.